controllers/datafoundationclient.go

```diff
@@ -115,6 +124,23 @@ func (r *dataFoundationClientReconciler) parseSpec(offering *v1alpha1.ManagedFus
 
 func (r *dataFoundationClientReconciler) reconcilePhases() (ctrl.Result, error) {
 	if !r.offering.DeletionTimestamp.IsZero() {
+		if found, err := r.hasOCSVolumes(); err != nil {
+			return ctrl.Result{}, err
+		} else if found {
+			r.Log.Info("Found consumer PVs using default storageclasses, cannot proceed with uninstallation")
```

Comment:
Why default? what if the customer has created an additional storageclass via a claim? And the message should be mentioning that we are addressing storageclasses that are provisioned by ODF. And don't use the "consumer" terminology anymore. 
```suggestion
			r.Log.Info("PVs provisioned by ODF  storage classes exists, cannot proceed with uninstallation")
```

Reply:
updated